### PR TITLE
Added weights, removed message when no item found

### PR DESCRIPTION
**Pull request by NicolaiAdil**

### CODE REVIEW

**Comment by EdvardSire** [comment]
Bro tf. Hvor er description? @NicolaiAdil

File: big_brother.py [hunk 1]
```diff
@@ -112,12 +110,9 @@ def use_image():
 
 
 if __name__ == "__main__":
-
     weight_path = "weights/best.pt"
     camera_url = "rtsp://big-brother:8554/ueye"
 
-
-    # model = torch.hub.load("ultralytics/yolov5", "yolov5s")
```
Comment (EdvardSire) [inline big_brother.py:120]: Sikker?

File: random_messages.py [hunk 2]
```diff
@@ -24,14 +24,14 @@ def custom_message(tool: str) -> str:
         "Social credit deducted :social_credit:",
         "Excecution date: 马公鸡",
         "Castration imminent",
-        "You are a disgrace to your family",
+        "You are a disgrace to your family"
```
Comment (EdvardSire) [inline random_messages.py:27]: Bra spott!

**Review by EdvardSire — CHANGES_REQUESTED** [review]
Kult

File: big_brother.py (hunk 1 above)
Comment (NicolaiAdil) [reply big_brother.py:120]: JA

File: random_messages.py (hunk 2 above)
Comment (NicolaiAdil) [reply random_messages.py:27]: Takk

**Review by EdvardSire — APPROVED** [review]

**Pull request merged**